lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1001,6 +1002,7 @@ sub auto_duplicate ($self, $args = {}) {
     $dup->{cluster_cloned} = {map { $_ => $clones->{$_}->{clone} } keys %$clones};
     $dup->{comments_created} = $args->{comments};
     log_debug("Job $job_id duplicated as $clone_id");
+    OpenQA::Events->singleton->emit_event('job_restart', data => {id => $clone_id});
```

Comment:
`auto_duplicate` is also called when doing a restart via API or web UI, and in that case the code emits an event already, see 
https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/WebAPI/Controller/API/V1/Job.pm#L821
So we would get two events.